Apps/W1/HybridGP/app/src/Migration/Customers/GPCustomerMigrator.codeunit.al

```diff
@@ -117,7 +117,7 @@ codeunit 4018 "GP Customer Migrator"
                     Sender.CreatePaymentTermsIfNeeded(Copystr(MigrationGPCustTrans.PYMTRMID, 1, 10), MigrationGPCustTrans.PYMTRMID, PaymentTermsFormula);
                 end;
                 Sender.SetGeneralJournalLinePaymentTerms(CopyStr(MigrationGPCustTrans.PYMTRMID, 1, 10));
-                Sender.SetGeneralJournalLineExternalDocumentNo(CopyStr(MigrationGPCustTrans.DOCNUMBR, 1, 20) + '-' + CopyStr(MigrationGPCustTrans.GLDocNo, 1, 14));
+                Sender.SetGeneralJournalLineExternalDocumentNo(CopyStr(MigrationGPCustTrans.DOCNUMBR, 1, 35));
```

Comment:
Without history I am not sure, but do we still need the CopyStr here. DOCNMBR is 22 characters which should be shorter than what BC accepts.

Reply:
Good point. Thinking about this more, we should probably Trim() the DOCNUMBR, and in that case should keep the CopyStr.